install/fo-install-pythondeps

```diff
@@ -159,7 +159,7 @@ if [[ $RUNTIME ]]; then
       # Include experimental dependencies
       su --login --whitelist-environment="http_proxy,HTTP_PROXY,https_proxy,HTTPS_PROXY" $TARGETUSER -c 'PYTHONPATH="$HOME/pythondeps" python3 -m pip install \
         --target=$HOME/pythondeps --no-input --upgrade \
-        spacy pandas scancode-toolkit==31.2.4 scanoss==1.5.1'
+        spacy pandas scancode-toolkit==31.2.4 scanoss==1.5.1 scikit-learn copyrightfpd'
```

Comment:
the package can now be updated to [safaa](https://pypi.org/project/safaa).

Reply:
Also, please tag a specific version of the dependencies to avoid issues in future and make code compliant.

Reply:
Updated the code to use safaa, currently the update is in a new commit, can squash the commits if necessary.